app/models/user.rb

```diff
@@ -1,5 +1,8 @@
 class User < ApplicationRecord
   
+  include Authenticator
+  attr_accessor :reset_token, :remember_token
```

Comment:
are we using this remember_token anywhere?

Reply:
done